config/config.go

```diff
@@ -59,6 +59,8 @@ type Configuration struct {
 	BlacklistedAcctMap map[string]bool
 	// Is publisher/account ID required to be submitted in the OpenRTB2 request
 	AccountRequired bool `mapstructure:"account_required"`
+	// True if we want to stop collecting account-specific metrics
```

Comment:
I think we want to put this in with the metrics config someplace. Probably in its own subobject, so we can add additional flags/metrics groups as needed.

Reply:
Done